src/interpreter.cpp

```diff
@@ -2344,25 +2351,29 @@ void nanny(DescriptorData *d, char *arg) {
                 echo_off(d);
                 d->idle_tics = 0;
                 STATE(d) = CON_PASSWORD;
+                if (!PLR_FLAGGED(d->character, PLR_LOGIN))
+                    SET_FLAG(PLR_FLAGS(d->character), PLR_LOGIN);
+                event_create(EVENT_PASS_TIMEOUT, pass_timeout, d, false, nullptr, PASS_TIMEOUT);
+                REMOVE_FLAG(PLR_FLAGS(d->character), PLR_LOGIN);
             }
         } else {
             /* No player loaded: we might make a new character. */
 
-            if (ispell_name_check(tmp_name)) {
+//            if (ispell_name_check(tmp_name)) {
```

Comment:
We're no longer instantly rejecting names in the dictionary?  (I approve.)

Reply:
Removed the commenting out - my bad!